tools/syz-cover/syz-cover.go

```diff
@@ -39,6 +39,7 @@ func main() {
 		flagConfig         = flag.String("config", "", "configuration file")
 		flagCoverAll       = flag.Bool("coverall", false, "set to true to use pc from Symbols")
 		flagModules        = flag.String("modules", "", "modules info obtained from /modules (optional)")
+		flagModuleFile     = flag.String("modulefile", "", "module file dump from /proc/modules (optional)")
```

Comment:
This is basically like `-modules`, but accepts a different input format, right?

Could we simplify the interface by trying to decode the file as JSON first and, if it fails, try to parse it like the raw `/proc/modules` output?

Or, alternatively, just rename it to `modules_raw` and highlight the expected format difference in flag descriptions. The `modulefile` name sounds confusing.